### PR TITLE
Fix camera crash

### LINKED ISSUE #4860: Qfield 3.1.3 crash after taking a picture when the phone has already been locked once.
### Describe the issue

Qfield crash when you take a picture with the Qfield camera but only after you have locked once the phone. See the reproduction steps for more details.

Maybe it is a duplicate of #4704 but the 3.1.3 with Qt 6.6.1 was supposed to fix it. Though, it seems to be a different bug because the 💥 only happen after locking and unlocking the phone, not after taking several pictures.

### Reproduction steps
1. Lauch a Qfield project with a layer with a picture field.
2. In the Qfield settings, deactivate the use of the native camera.
3. Add an entity and take a picture, validate it, everything is fine.
4. Lock your phone with its lock buttons.
5. Unlock it
6. Add a entity, add a picture in it. As soon as you will touch the big white button, qfield will 💥.

### Desktop (please complete the following information)
 - OS: Windows 10
 - QGIS Version: 3.34.1
 - QFieldSync Version: 4.6.1

### Mobile (please complete the following information)
 - Device: Samsung S20 FE 5G
 - OS: Android 13
 - QField version: 3.1.3 (14a225)

### Additional information
* Problem started happening recently, didn't happen in an older version of QField: I am not sure.
* Problem can be reliably reproduced, doesn't happen randomly: Yes
* Problem happens with all files and projects, not only some files or projects: Yes

### LINKED ISSUE #4884: Qfield Crashes, when using the QR code scanner Feature for more than one Feature
### Describe the issue
Qfield is crashing when the (QR) code scan tool (opened by the 3 dots beside the Field to fill in value) is opened.
Happens only when there was a Picture taken in advance and/or a Code scanned in advance

### Reproduction steps
1. Create and open a project, with one field for picture, one for text
2. Start digitizing/creating new features
3. Add a picture to the entry formula and scan a code, respective to fields
4. Save feature
5. Start adding a new feature and try to read a code

### Expected behavior
Qfield does read qr codes, when new feature is added

### Observed behavior
Qfield crashes when code reader is opened. Briefly, the shown camera preview window is freezing. Then immediate crash

### Desktop (please complete the following information)
 - OS: Windows
 - QGIS Version 3.30.2

### Mobile (please complete the following information)
 - Device: Samsung Galaxy A52s 5G
 - OS: Android 13
 - QField version: 3.1.3 (14a255 v3.1.3) Qt 6.6.1

### Additional information
* Problem started happening recently, didn't happen in an older version of QField: Unknown (Just started using the QR code Scan tool)
* Problem can be reliably reproduced, doesn't happen randomly: Yes
* Problem happens with all files and projects, not only some files or projects: Yes

### PR DESCRIPTION
**Pull request by m-kuhn**
With this PR, Qt for android is being built by CI.
This allows us to integrate fixes on Qt without having them in an official release, first time done here for a camera crash.
The new way cannot be properly built for x86 devices, we therefore retire x86 android builds. There are very few devices on this architecture (it accounts to 0.01% of all installations, amount decreasing).

Fixes #4860 
Fixes #4884

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 294377247d9af98ec25ffdd40eb693905352bbbf: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4893-d943f3bade94295506cd2ace00328722357f897f-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4893-d943f3bade94295506cd2ace00328722357f897f-arm-neon-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4893-d943f3bade94295506cd2ace00328722357f897f-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4893-d943f3bade94295506cd2ace00328722357f897f-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4893-d943f3bade94295506cd2ace00328722357f897f-arm64-android.apk)
- [all access arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4893-d943f3bade94295506cd2ace00328722357f897f-arm-neon-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4893-d943f3bade94295506cd2ace00328722357f897f-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4893-d943f3bade94295506cd2ace00328722357f897f-x86-android.apk)
</details>

File: vcpkg/overlay/qtmultimedia/portfile.cmake [hunk 1]
```diff
@@ -64,7 +64,7 @@ endif()
 if("ffmpeg" IN_LIST FEATURES)
     # Note: Requires pulsadio on linux and wmfsdk on windows
     list(APPEND FEATURE_OPTIONS "-DINPUT_ffmpeg='yes'")
-    if(VCPKG_TARGET_IS_WINDOWS OR VCPKG_TARGET_IS_OSX)
+    if(VCPKG_TARGET_IS_WINDOWS OR VCPKG_TARGET_IS_OSX OR VCPKG_TARGET_IS_ANDROID)
```
Comment (m-kuhn) [inline vcpkg/overlay/qtmultimedia/portfile.cmake:67]: TODO: upstream

Comment (nirvn) [reply vcpkg/overlay/qtmultimedia/portfile.cmake:67]: Absolutely

**Comment by nirvn** [comment]
Superseded by #4898

**Pull request closed without merging**